Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -601,45 +565,20 @@ CGPathRef CGPathRetain(CGPathRef path) {
 }
 
 /**
- @Status Caveat
- @Notes transform property not supported
+@Status Stub
```

Comment:
There will be a document on OneNote with the status of any APIs that have changed status. It may be helpful to look at the API status snapshots on develop to double check any APIs in question.

---
In reply to: [83132884](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83132884) [](ancestors = 83132884)